### ISSUE TITLE
Add NNODE example

### ISSUE THREAD

**Issue by sdwfrost** [epirecipes/sir-julia#75]
Currently stuck on adding an NNODE - an important prerequisite for using NeuralPDE.jl to fit PDE models. See [https://discourse.julialang.org/t/constraints-in-neuralpde-nnode/92723/2](https://discourse.julialang.org/t/constraints-in-neuralpde-nnode/92723/2) for a toy example that fails.

**Comment by ChrisRackauckas** [comment]
Yeah the simple MLP neural networks are usually not sufficient. I'm not sure if that's just a documentation issue or something we can handle more easily within the package itself, given that NNODE is a constrained form. Maybe it should default to an architecture we know well and only give options for layer sizes.
